src/sdk/namespace_fs.js

```diff
@@ -682,7 +682,50 @@ class NamespaceFS {
         const fs_account_config = this.set_cur_fs_account_config(object_sdk);
         await this._load_bucket(params, fs_account_config);
 
+        const can_use_temp_file = Boolean(fs_account_config.backend === 'GPFS' || nb_native().fs.gpfs);
```

Comment:
The tests fail because this needs to be &&, not ||.
And since some of our tests set backend type to GPFS this fails to upload because we don't have the library in CI.
(search 'GPFS' in src/test/unit_tests/test_bucketspace.js)